src/appmetrics.cpp

```diff
@@ -528,8 +544,11 @@ NAN_METHOD(sendControlCommand) {
     }
 
     if (info[0]->IsString() && info[1]->IsString()) {
-        String::Utf8Value topicArg(info[0]->ToString());
-        String::Utf8Value commandArg(info[1]->ToString());
+        Isolate* isolate = v8::Isolate::GetCurrent();
+        // String::Utf8Value topicArg(isolate, Nan::To<String>(info[0]).ToLocalChecked());
+        Nan::Utf8String topicArg(info[0]);
+        // String::Utf8Value commandArg(isolate, Nan::To<String>(info[1]).ToLocalChecked());
+        Nan::Utf8String commandArg(info[1]);
```

Comment:
I think this is one of the places that breaks Node 6. Is there any reason we can't use if NODE_VERSION_AT_ to detect Node 6 here? We know the code used to work on Node 6, we know what to #ifdef in and out and travis can keep the testing in parallel.

Reply:
I'd be surprised if this breaks Node 6 as `nan` should still work with Node 6.